Classes/ViewHelpers/ImageViewHelper.php

```diff
@@ -536,10 +536,13 @@ protected function getFrontendController(): ?TypoScriptFrontendController
     protected function applyProcessingInstructions($image, array $processingInstructions): ProcessedFile
     {
         if (($processingInstructions['fileExtension'] ?? '') === 'webp'
-            && ($this->checks['lossless'] ?? false)
             && $image->getExtension() !== 'webp'
         ) {
-            $processingInstructions['additionalParameters'] = '-define webp:lossless=true';
+            if ($this->checks['lossless'] ?? false) {
+                $processingInstructions['additionalParameters'] = '-define webp:lossless=true';
+            } else {
+                $processingInstructions['additionalParameters'] = '-quality 85';
```

Comment:
a) this is a feature, not a task :) 
b) I recommend using the jpegQuality setting

    $jpegQuality = MathUtility::forceIntegerInRange($GLOBALS['TYPO3_CONF_VARS']['GFX']['jpg_quality'], 10, 100, 85);

Reply:
Good points. I will use the jpg_quality setting for this 👍

Reply:
I used the jpgQuality as suggested.